packages/turf-line-chunk/index.js

```diff
@@ -1,15 +1,14 @@
 var lineSliceAlong = require('@turf/line-slice-along');
 var lineDistance = require('@turf/line-distance');
 var featureCollection = require('@turf/helpers').featureCollection;
-var featureEach = require('@turf/meta').featureEach;
-var flatten = require('@turf/flatten');
+var flattenEach = require('@turf/meta').flattenEach;
 
 /**
  * Divides a {@link LineString} into chunks of a specified length.
  * If the line is shorter than the segment length then the original line is returned.
  *
  * @name lineChunk
- * @param {FeatureCollection|Feature<LineString|MultiLineString>} featureIn the lines to split
+ * @param {FeatureCollection|Feature|GeometryCollection<LineString|MultiLineString>} featureIn the lines to split
```

Comment:
For the JSDocs I've been using `Geometry` to describe all `Geometry Collection` & `Geometry Objects`.